mscore/editpitch.cpp

```diff
@@ -65,16 +60,14 @@ void EditPitch::hideEvent(QHideEvent* ev)
 
 void EditPitch::accept()
       {
-      int   col = tableWidget->currentColumn();
-      int   row = tableWidget->currentRow();
-
-      // topmost row contains notes for 9-th MIDI octave (numbered as '8')
-      done( (9 - row)*12 + col);
+      on_tableWidget_cellDoubleClicked(tableWidget->currentRow(), tableWidget->currentColumn());
       }
 
 void EditPitch::on_tableWidget_cellDoubleClicked(int row, int col)
       {
-      done( (9 - row)*12 + col);
+      // topmost row contains notes for 10-th MIDI octave (numbered as '9')
+      int pitch = (tableWidget->rowCount()-1 - row) * 12 + col;
+      done((pitch > 127)? 127: pitch);
```

Comment:
shouldn't happen, with those fields above G9/127 being disabled and unselectable, but better safe than sorry...

Reply:
127 should be ok.